### DISCUSSION TITLE
UserLevel 관련 회의 필요

### DISCUSSION THREAD

**Discussion by jiwon11** [MetaAir2023/WebRPG#23] [General]
# 레벨의 한계치
# 레벨업을 위한 경험치 필요량
  - 경험치 필요량 고정 혹은 증감
  - 고정이라면 얼마 고정, 증감이라면 증감폭을 얼마나 할 것인지.
# 레벨에 따른 능력치 증가
  - 공격력, 체력
  - 증감치는 얼마나 할 것인지, 레벨에 따라 증감치에 차이를 둘 것인지.

**Comment by max0901** [comment]
경험치 고정에 경험치량은 10마리 정도의 몬스터를 잡으면 레벨업을 할수있게 경험치 고정량을 정하면 될꺼같습니다
레벨에 따른 능력치는 레벨 *10으로 해서 증가 했으면 좋겠습니다

**Comment by ahnichan** [comment]
레벨 한계 300, 경험치 필요량은 레벨이 올라갈수록 증가, 증가폭은 레벨 당 100씩 증가 
공격력,체력 레벨 당 일괄 100씩 증가

**Reply by jiwon11** [reply]
경험치 증가폭이 레벨 당 100씩 증가하는것은 100 -> 200 -> 300 씩 증가한다는 뜻인가요??

**Comment by aicompose** [comment]
체력바 로직을 그대로 사용해서 체력바가 풀칸이 되면 레벨업, 레벨업을 할 수록 래밸업 난이도가 어렵게 하는 게 좋을 것 같긴한데 구현 난이도가 높을 것 같아서  1 -> 2,  2- >3 차이를 안 두고 레벨업당 * 공격력(%) 이런식으로 하는 게 좋지 않을까 생각됩니다

**Comment by madirony** [comment]
1. 경험치 필요량 증감 필요
    - 상위 레벨의 몬스터 -> 경험치 및 보상이 클 것으로 예상
    -  증감폭은 현재 레벨에 특정 정수(10 혹은 100 등)를 곱하면 좋겠음
2. 레벨에 따른 증감치는 고정으로 하고, 공격력은 10 정도, 체력은 100정도 증가

**Comment by jiwon11** [comment]
레벨업 시 스탯 증가량은 고정 : 공격력 = 레벨 *100 /  체력= 레벨*100

**Comment by happy-baechuchu** [comment]
경험치 필요량 레벨에 따라 50 단위로 증가?
레벨업 시 공격력, 체력 고정값 증가

**Comment by jiwon11** [comment]
레벨 별 레벨업 필요 경험치 = 현재 레벨 * 상수 *  10
